front/js/productCountInput.es6

```diff
@@ -28,7 +26,7 @@
         mediator.publish('onProductsCountDown', $(this).siblings(DOM.countInput));
       });
 
-    $(DOM.countInput).on('input', productPriceSum);
+    $(DOM.countInput).on('input', event => changeSum(event, event.target));
```

Comment:
@artemiy312 , check this string plz. I'm not sure, that passing the first arg `event` is idiomatic for the changeSum function